test/mailers/business_mailer_test.rb

```diff
@@ -0,0 +1,12 @@
+require "test_helper"
+
+class BusinessMailerTest < ActionMailer::TestCase
+  test "subscribed" do
+    BusinessMailer.with(business:).subscribed.deliver_later
+    assert_enqueued_emails 1
+  end
+
+  def business
+    businesses(:subscriber)
+  end
+end
```

Comment:
We can delete this test. The `verify_mailer_previews` job ensures that none of them break.